saidump/main.cpp

```diff
@@ -0,0 +1,11 @@
+#include "SaiDump.h"
+
+using namespace syncd;
+
+int main(int argc, char **argv)
+{
+    SWSS_LOG_ENTER();
+    SaiDump m_saiDump;
```

Comment:
No need to this me prefixes m, this is local var

Reply:
Fixed.